web/lang/en_gb.php

```diff
@@ -1016,6 +1016,18 @@ function zmVlang( $langVarArray, $count )
       for new images. In this case, it is safe to use the field.
       '
 	),
+	'OPTIONS_LINKED_MONITORS' => array(
+    'Help' => '
+      This field allows you to select other monitors on your system that act as 
+      triggers for this monitor. So if you have a camera covering one aspect of 
+      your property you can force all cameras to record while that camera 
+      detects motion or other events.  You can either directly enter a comma 
```

Comment:
I don't think you can enter a comma separated list.  The text input is a filter for the dropdown.  There is a text input underlying the widget but it isn't accessible.  It was in older versions of ZM, but since 1.32 we are using this fancy selection widget.

Reply:
cool. I fixed the typo and changed the text to reflect the new widget